py3status/modules/wifi.py

```diff
@@ -66,16 +68,34 @@ class Py3status:
     cache_timeout = 10
     device = 'wlan0'
     down_color = 'bad'
-    format_down = 'W: down'
-    format_up = 'W: {bitrate} {signal_percent} {ssid}'
+    format = DEFAULT_FORMAT
     round_bitrate = True
     signal_bad = 29
     signal_degraded = 49
     use_sudo = False
 
     def post_config_hook(self):
+        # DEPRECATION WARNING
+        format_down = getattr(self, 'format_down', None)
+        format_up = getattr(self, 'format_up', None)
+        format = getattr(self, 'format', None)
```

Comment:
why not just use `self.format` it is always there?

this is now looking like it works, I'll test it once I get some time to spare